### ISSUE TITLE
Fix scrolling, windows devices cannot scroll down easily

### ISSUE THREAD

**Issue by yassirog** [fdnd-agency/fdnd#67]
Windows machines can't scroll through the page, they need to click anywhere and use arrows on their keyboards to scroll.
